src/types/config.ts

```diff
@@ -89,10 +89,19 @@ export type DashboardConfig = {
    */
   showEntryPoint: boolean;
 
+  /** Name of the RPC method to get a valid auth token */
+  authTokenRPCMethod: string;
+
   /**
-   * Name of the RPC method to call in embedded frame on entry point activation.
+   * Entry point for the dashboard, where the first request needs to happen to
+   * get authenticated.
    */
-  entryPointRPCMethod: string;
+  entryPoint: {
+    path: string;
+  };
```

Comment:
This "shape" might be a bit weird. I tried to use the same structure we use for `APICallInfo` in LMS, but we don't usually expose those to the client 🤷🏼‍♂️ 

Maybe we could just go with `entryPointPath: string`.

Reply:
This pops two questions:

* Are you agreeing with my suggestion to actually flatten this prop to become a simple string instead of an object?
* This is in fact a path, not a full URL, in that's what the rest of the code assumes. See [how it is used](https://github.com/hypothesis/client/pull/6323/files/5dc7567356003dbb289e4deb31c7ea4da4fe03de#diff-01158a460efb6249246bf87fbd78becae3202b3785564697652fdfc62cf2bbb1R37), by prepending the RPC origin.
  Would you suggest we actually return an already formed URL here and avoid using `rpc.origin` there?

Reply:
> Are you agreeing with my suggestion to actually flatten this prop to become a simple string instead of an object?

Yes. The reason the LMS app uses an object is to be able to specify things like the HTTP method and data to post with the request as a JSON body. Here we just want a plain URL.

> This is in fact a path, not a full URL, and that's what the rest of the implementation assumes. See [how it is used](https://github.com/hypothesis/client/pull/6323/files/5dc7567356003dbb289e4deb31c7ea4da4fe03de#diff-01158a460efb6249246bf87fbd78becae3202b3785564697652fdfc62cf2bbb1R37), by prepending the RPC origin.
Would you suggest we actually return an already formed URL here and avoid using rpc.origin?

I see. I think a full URL will be better here.

Reply:
Thanks!